src/client/components/EditorWYSIWYG/EditorWYSIWYG.tsx

```diff
@@ -68,6 +76,11 @@ const EditorWYSIWYG: React.FC<Props> = (props: Props) => {
     [onChange]
   )
 
+  useEffect(() => {
+    if (joditEditor) joditEditor.setReadOnly(disabled)
+    if (joditEditor?.container) joditEditor.toolbarContainer.classList.toggle('toolbar-disabled', disabled)
+  }, [disabled, joditEditor])
+
```

Comment:
the actual fix

it was possible also to change the css prop to hide toolbar but it would leave the dashed border, so added css class and set border to unset

Reply:
i will try this

Reply:
good suggestion and works as expected

(lets verify in review anyway)